create-k8s-chained-sessions.sh

```diff
@@ -6,63 +6,92 @@
 # Do not run this script more than once without resetting your Leapp
 # instance beforehand.
```

Comment:
Does this warning still apply now that this is idempotent?

Reply:
No, I can remove it.